pwa/app/components/tba/match/matchRows.tsx

```diff
@@ -94,110 +94,114 @@ export function MatchRow({
   const isPlayed =
     match.alliances.red.score !== -1 && match.alliances.blue.score !== -1;
 
+  /* Desktop: 1x11 grid, Mobile: 2x6 grid */
   return (
-    <div>
-      {/* Desktop: 1x11 grid, Mobile: 2x6 grid */}
+    <div
+      className="mx-auto grid w-full max-w-6xl
+        grid-cols-[2.5em_7em_repeat(4,1fr)] grid-rows-[2.5em_2.5em] gap-0.5
+        text-sm xl:grid-cols-[2.5em_7em_repeat(9,1fr)] xl:grid-rows-1"
+    >
+      {/* Play Button */}
       <div
-        className="mx-auto grid w-full max-w-6xl
-          grid-cols-[2.5em_7em_repeat(4,1fr)] grid-rows-[2em_2em] gap-x-1
-          text-sm xl:h-8 xl:grid-cols-[2.5em_7em_repeat(9,1fr)] xl:grid-rows-1"
+        className="row-span-2 flex items-center justify-center rounded-tl-lg
+          xl:col-span-1 xl:row-span-1 xl:rounded-l-lg"
       >
-        {/* Play Button */}
-        <div
-          className="row-span-2 flex items-center justify-center rounded-tl-lg
-            bg-gray-100 xl:col-span-1 xl:row-span-1 xl:rounded-l-lg"
+        {maybeVideoURL && (
+          <Link to={maybeVideoURL} className="mx-2">
+            <PlayCircleIcon />
```

Comment:
@ofekashery do you have any suggestions on whether this link to the first match video is a good UX pattern? Personally, I don't think playing lottery with the first video is great, especially considering the match modal loads so quickly and shows all the videos. Maybe the entire row can be one big button that opens the modal? Curious to hear your thoughts.

Reply:
Maybe create a temporary playlist with all match records like our Watch all matches button?